pwndbg/commands/context.py

```diff
@@ -278,21 +279,23 @@ def context_backtrace(frame_count=10, with_banner=True):
     return result
 
 
-def context_args():
+def context_args(with_banner=True):
     result = []
 
-    ##################################################
-    # DISABLED FOR NOW, I LIKE INLINE DISPLAY BETTER
-    ##################################################
-    # # For call instructions, attempt to resolve the target and
-    # # determine the number of arguments.
-    # for arg, value in pwndbg.arguments.arguments(pwndbg.disasm.one()):
-    #     code   = False if arg.type == 'char' else True
-    #     pretty = pwndbg.chain.format(value, code=code)
-    #     result.append('%-10s %s' % (arg.name+':', pretty))
-    # if not result:
-    #         return []
-    # result.insert(0, pwndbg.ui.banner("arguments"))
+    # For call instructions, attempt to resolve the target and
+    # determine the number of arguments.
+    one = pwndbg.disasm.one()
+    for arg, value in pwndbg.arguments.get(one):
+        code   = False if arg.type == 'char' else True
+        pretty = pwndbg.chain.format(value, code=code)
+        result.append('%8s%-10s %s' % ('', N.argument(arg.name) + ':', pretty))
```

Comment:
`%8s` matches `''` - maybe we should just do '        %-10s %s'`?

Reply:
8 spaces look more weird then having %8s, i think its more obvious that there is a space of 8 something